src/components/common/navbar.tsx

```diff
@@ -36,16 +39,27 @@ const Navbar: FC<Props> = ({
     setOpenSearchField(false);
   };
 
+  const onSearch = (event: KeyboardEvent<HTMLInputElement>) => {
```

Comment:
This method seems duplicated.
Can you consider moving this to util and re-use it ?

Reply:
That method is not duplicated. In order to reuse the searchbar I moved the onSearch function to its parent component. So that we can implement our custom search functionality. In navbar the search is implemented for order search